src/main/java/com/minecolonies/entity/ai/citizen/fisherman/Pond.java

```diff
@@ -11,7 +11,9 @@
  */
 public final class Pond
 {
-    private static final String TAG_LOCATION = "Location";
+    private static final String TAG_LOCATION                  = "Location";
+    private static final int    WATER_POOL_WIDTH_REQUIREMENT  = 6;
+    private static final int    WATER_POOL_HEIGHT_REQUIREMENT = 3;
 
     private BlockPos location;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'location' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)